scripts/hyperkit.js

```diff
@@ -0,0 +1,20 @@
+// This downloads hyperkit, builds it, and puts the binary in the right place.
+
+const fs = require('fs');
+const os = require('os');
+const { spawn, spawnSync } = require('child_process');
+
+fs.mkdirSync("./resources/" + os.platform(), { recursive: true });
+
+// The version of hyperkit to build
+let ver = "v0.20210107";
+
+// TODO: rewrite to remove callbackitis
+spawn('git', ['clone', '--depth', '1', '--branch', ver, "https://github.com/moby/hyperkit.git"], { cwd: '/tmp/' }).on('exit', () => {
+  spawn('make', [], { cwd: '/tmp/hyperkit/' }).on('exit', () => {
+    spawn('cp', ['-f', '/tmp/hyperkit/build/hyperkit', process.cwd() + '/resources/' + os.platform() + '/hyperkit']).on('exit', () => {
+      spawnSync('rm', ['-rf', '/tmp/hyperkit']);
```

Comment:
Should probably use [`fs.rmdir()`](https://nodejs.org/api/fs.html#fs_fs_rmdir_path_options_callback) here.  Also, we should avoid sync things (though I suspect you did it here due to too much callbacks; [`util.promisify()`](https://nodejs.org/api/util.html#util_util_promisify_original) would probably help).

Reply:
It looks like rmdir with recursive set to true is what I would want... but it's deprecated and will throw an error in the future. It looks like using rm would be better.

Reply:
specifically, the docs say...
> Setting recursive to true results in behavior similar to the Unix command rm -rf: an error will not be raised for paths that do not exist, and paths that represent files will be deleted. The permissive behavior of the recursive option is deprecated, ENOTDIR and ENOENT will be thrown in the future.